website/static/css/style.css

```diff
@@ -337,6 +337,12 @@ a {
     margin-top: 4.8px;
 }
 
+/* Disable pointer events in page changing */
+.disabled {
```

Comment:
My concern is that this overrides a built-in Bootstrap class. @chennan47 Why is this rule necessary? What does it provide that Bootstrap does not?